tasks/download_kubeconfig.yml

```
@@ -0,0 +1,21 @@
+---
+
+- name: Download k3s kubeconfig to localhost
+  ansible.builtin.fetch:
+    src: /etc/rancher/k3s/k3s.yaml
```

```suggestion
    src: /{{ k3s_config_dir }}/k3s.yaml
```

https://github.com/PyratLabs/ansible-role-k3s/blob/80e4debcd4a7b4c8d596a380743ca1017acad43d/vars/main.yml#L105-L106